>>> from exceptional import convert
>>> convert("fail".split())
Conversion error: KeyError('fail')
-1

